src/commands/install-docker-credential-helper.yml

```diff
@@ -23,74 +23,7 @@ parameters:
 steps:
   - run:
       name: Install Docker credential helper
-      command: |
-        HELPER_NAME="<<parameters.helper-name>>"
-        if [ -z "${HELPER_NAME}" ]; then
-          if [ -n "$(uname | grep "Darwin")" ]; then
-            HELPER_NAME="osxkeychain"
-          else
-            HELPER_NAME="pass"
-          fi
-        fi
-
-        HELPER_FILENAME="docker-credential-${HELPER_NAME}"
-
-        if which "$HELPER_FILENAME" > /dev/null 2>&1; then
-          echo "$HELPER_FILENAME is already installed"
-          exit 0
-        fi
-
-        SUDO=""
-        if [ $(id -u) -ne 0 ] && which sudo > /dev/null 2>&1 ; then
-          SUDO="sudo"
-        fi
-
-        # Create heredoc template here due to tab indentation issue
-        GPG_TEMPLATE=$(mktemp gpg_template.XXXXXX)
-        cat > $GPG_TEMPLATE \<<-EOF
-        Key-Type: RSA
-        Key-Length: 2048
-        Name-Real: CircleCI Orb User
-        Name-Email: circleci-orbs@circleci.com
-        Expire-Date: 0
-        %no-protection
-        %no-ask-passphrase
-        %commit
-        EOF
-
-        if [ "$HELPER_FILENAME" = "docker-credential-pass" ]; then
-          # Install pass which is needed for docker-credential-pass to work
-          $SUDO apt-get update --yes && $SUDO apt-get install gnupg2 pass --yes
-
-          # Initialize pass with a gpg key
-          gpg2 --batch --gen-key "$GPG_TEMPLATE"
-
-          FINGERPRINT_STRING=$(gpg2 \
-            --list-keys --with-fingerprint --with-colons \
-            circleci-orbs@circleci.com | \
-            grep fpr)
-          # use colon delimiters to separate fingerprint
-          arrFINGERPRINT=(${FINGERPRINT_STRING//:/ })
-          FINGERPRINT=${arrFINGERPRINT[-1]}
```

Comment:
This wasn't working super well and shellcheck was complaining. I ended up replacing it with grep:

https://github.com/CircleCI-Public/docker-orb/pull/116/files#diff-f7370f8e2b70d398d33f378fdba649d3cdbd8837764e52bd3cab81c104294277R49